posts/inside-rust/2020-07-08-lang-team-path-to-membership.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "Lang team design meeting: path to membership"
+author: Niko Matsakis
+description: "Lang team design meeting: path to membership"
+team: the lang team <https://www.rust-lang.org/governance/teams/lang>
+---
+
+Today the [lang team] design meeting was on the topic of the "path to
+membership". This blog post gives a brief summary; you can also read
+the [minutes] or view the [recording].
+
+[minutes]: https://github.com/rust-lang/lang-team/blob/master/design-meeting-minutes/2020-07-08-lang-team-path-to-membership.md
+[lang team]: https://www.rust-lang.org/governance/teams/lang
+[recording]: https://youtu.be/SeH-hZgDG1Y
+
+The premise of the meeting was that the lang team has never had a
+particularly clear *path to membership* -- i.e., it's been hard to say
+exactly what are the kinds of steps that one should be taking if you
+would like to become a member of the lang team. However, with the
+shift to [major change proposals] and in particular [project groups],
+we're starting to see what such a path looks like. Simultaneously,
+that same shift is helping to reveal that overall lang team bandwidth
+remains low, and that we could use to grow the team -- so thinking
```

Comment:
I'm having trouble understanding this sentence, perhaps changed mid-way through writing?

Reply:
meh, I'm just going to cut the sentence.